src/main/java/org/dcsa/tnt/controller/EventController.java

```diff
@@ -1,39 +1,30 @@
 package org.dcsa.tnt.controller;
 
 import lombok.RequiredArgsConstructor;
-import org.dcsa.core.events.controller.AbstractEventController;
+import org.dcsa.core.controller.ExtendedBaseController;
 import org.dcsa.core.events.model.Event;
-import org.dcsa.core.events.util.ExtendedGenericEventRequest;
-import org.dcsa.core.extendedrequest.ExtendedRequest;
-import org.dcsa.tnt.service.EventService;
-import org.springframework.http.HttpStatus;
+import org.dcsa.core.events.service.GenericEventService;
 import org.springframework.http.MediaType;
-import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.server.ResponseStatusException;
-import reactor.core.publisher.Mono;
-import javax.validation.Valid;
+
+import java.util.UUID;
 
 @RestController
 @RequiredArgsConstructor
 @RequestMapping(value = "events", produces = {MediaType.APPLICATION_JSON_VALUE})
-public class EventController extends AbstractEventController<EventService, Event> {
+public class EventController extends ExtendedBaseController<GenericEventService, Event, UUID> {
```

Comment:
If you make this change some of the methods we have disabled (Intentionally) in `AbstractEventController` will be enabled again. For example: update, delete & deleteById for events. You will need include that change too.

Reply:
Should be fixed now.